### ISSUE TITLE
Please add Documentation

### ISSUE THREAD

**Issue by codebeaulieu** [hannesbarbez/BarbezDotEu.StockTwits#6]
**Is your feature request related to a problem? Please describe.**
I found your GitHub project by searching StockTwits in Nuget, however I don't know what this library does.

**Describe the solution you'd like**
At a minimum, high-level documentation that describes how to use the library.

**Describe alternatives you've considered**
Browsed the code, still not sure.

**Comment by hannesbarbez** [comment]
This is an unofficial, modern, very much work-in-progress client for StockTwits APIs.

Will add documentation when I have the time. For now, it's mostly for my own use.

**Comment by codebeaulieu** [comment]
> This is an unofficial, modern, very much work-in-progress client for StockTwits APIs.

Yeah, I've read that. I'm wondering if this requires proper API keys or if it leverages a regular login and password from any account.

**Comment by hannesbarbez** [comment]
Reopening due to concrete topic.
